src/main/java/org/jboss/forge/arquillian/api/AlgeronSetupFacet.java

```diff
@@ -0,0 +1,78 @@
+package org.jboss.forge.arquillian.api;
+
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.forge.addon.dependencies.builder.DependencyBuilder;
+import org.jboss.forge.addon.facets.constraints.FacetConstraint;
+import org.jboss.forge.addon.projects.facets.DependencyFacet;
+import org.jboss.forge.addon.projects.facets.MetadataFacet;
+
+@FacetConstraint(TestFrameworkFacet.class)
```

Comment:
Forge attempts to install this facet if not found. As there are multiple implementations (two), your test needs to install a facet that implements TestFrameworkFacet before installing this facet